modules/tinymce/src/themes/silver/main/ts/ui/dialog/CustomEditor.ts

```diff
@@ -23,6 +24,16 @@ export const renderCustomEditor = (spec: CustomEditorSpec): SimpleSpec => {
   });
 
   const initialValue = Singleton.value<string>();
+  const focusBehaviour = !isOldCustomEditor(spec) && spec.settings?.focusSelector ? [
```

Comment:
I think the `settings` was supposed to be a anything object. I think this should go into the `CustomEditorNew` structure instead as an optional thing or even make it a function like `onFocus: () => void` that way you can do anything when the component gets focus.

- [ ] Move this to the actual spec so that it's properly validated and documented in the types